src/components/Button/Button.css

```diff
@@ -0,0 +1,103 @@
+.App{
+    display: flex;
+    flex-direction: row;
+}
+.App button{
```

Comment:
Для стилизации кнопки используй класс .button. Не используй тег button - это плохая практика
```suggestion
.button{
```